### PR TITLE
Week2

### PR DESCRIPTION
**Pull request by Radhikajram**

### CODE REVIEW

File: homework/index.js [hunk 1]
```diff
@@ -30,15 +29,142 @@
...
-      } else {
-        createAndAppend('pre', root, { text: JSON.stringify(data, null, 2) });
+  // Load the Contributors information into  section2 div element.
+
+  function addContributors(contribution) {
+    const divElement = [];
+
+    divElement.push('<div id="header">', `${'<h3>'} Contributions: ${'</h3>'} ${'</div>'}`);
+
+    // eslint-disable-next-line no-restricted-syntax
```
Comment (grod220) [inline homework/index.js]: Please remove all eslint-disable comments

File: homework/index.js [hunk 2]
```diff
@@ -30,15 +29,142 @@
...
+    // push all the HYP repo names to sort array.
+
+    // eslint-disable-next-line no-restricted-syntax
+    for (const repo in userRepo) {
+      if (Object.prototype.hasOwnProperty.call(userRepo, repo)) {
+        sortRepoName.push(userRepo[repo].name);
+      }
+    }
+
+    // sort the repo name using sort function and localeComapare for uppercase and lowercase sorting.
```
Comment (grod220) [inline homework/index.js:100]: Really like the comments that add context and decision thinking 👍

File: package.json [hunk 3]
```diff
@@ -9,7 +9,7 @@
   "author": "Jim Cramer",
   "license": "CC-BY-4.0",
   "devDependencies": {
-    "eslint": "^5.9.0",
+    "eslint": "^5.16.0",
```
Comment (grod220) [inline package.json:12]: Why was this updated?

**Review by grod220 — CHANGES_REQUESTED** [review]
Really solid, just had a few comments before the approval

File: package.json (hunk 3 above)
Comment (Radhikajram) [reply package.json:12]: when i run npm install it got automatically modified.

**Review by grod220 — APPROVED** [review]

**Pull request merged**